packages/core/src/utils/collect-updates/collect-updates.ts

```diff
@@ -17,7 +17,7 @@ import { makeDiffPredicate } from './lib/make-diff-predicate';
  * @param {UpdateCollectorOptions} commandOptions
  */
 export function collectUpdates(filteredPackages: Package[], packageGraph: PackageGraph, execOpts: ExecOpts, commandOptions: UpdateCollectorOptions, gitDryRun = false) {
-  const { forcePublish, conventionalCommits, conventionalGraduate, excludeDependents } = commandOptions;
+  const { forcePublish, conventionalCommits, conventionalGraduate, excludeDependents, isIndependent } = commandOptions;
```

Comment:
not sure why you added the `isIndependent` since we already have something similar on the project, for example on the first line of the initialize we have this check
```ts
if (!this.project.isIndependent()) {
      this.logger.info('current project version', this.project.version ?? '');
}
```
can't we use what's already available?

Reply:
I do on the command level but I have no access to `project` here.

Reply:
ok it's the separate util file, fine by me then